database/20220810_000000_Add_UserAccount_SoftcorePoints_Key.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE UserAccounts ADD KEY `RASoftcorePointsUntracked` (`RASoftcorePoints`,`Untracked`) USING BTREE;
```

Comment:
The softcore rank query was very slow without this. It's basically a clone of the `RAPointsUntracked` KEY.

Reply:
Executed on production